### ISSUE TITLE
Implement Block/ Unblock in the Block/ Unblock Page 

### ISSUE THREAD

**Issue by tasneemkoushar** [PalisadoesFoundation/talawa-admin#483]
**Is your feature request related to a problem? Please describe.**
We occasionally need to ban users who begin spamming your organization. With our `talawa-admin` portal, there is already a dedicated page `Block/ Unblock` menu in Navbar. Presently, the block functionality is functioning well, however, after an admin blocks a user, the user simply disappears and cannot be unblocked. Indeed, the admin also lacks a function that would allow them to view the list of blocked users.
![9](https://user-images.githubusercontent.com/58592306/219936951-f9ae8854-c6c1-4f84-871f-aacd02ef27f3.PNG)

**Describe the solution you'd like**
+ An admin should be able to block members.
+  Admin should be able to get a list of blocked members.
+  Admin should be able to unblock blocked members

**Approach to be followed (optional)**
We want to minimize changes to the existing design. Hence no new CSS, no new responsive design, just the enabling of previously disabled or incorrectly implemented features.

**Comment by Manoramsharma** [comment]
Hi, @tasneemkoushar I would like to work on this issue.

**Comment by kb-0311** [comment]
@tasneemkoushar @anwersayeed pls checkout this conversation https://thepalisadoes-dyb6419.slack.com/archives/CSWH4PN0M/p1677074629933039?thread_ts=1676728476.140449&cid=CSWH4PN0M

**Comment by tasneemkoushar** [comment]
> @tasneemkoushar @anwersayeed pls checkout this conversation https://thepalisadoes-dyb6419.slack.com/archives/CSWH4PN0M/p1677074629933039?thread_ts=1676728476.140449&cid=CSWH4PN0M

Okay

**Comment by palisadoes** [comment]
@Manoramsharma Unassigning due to inactivity. We need to give others a chance.

**Comment by prathamesh-mutkure** [comment]
Okay seems like Block/Unblock has been implemented in the backend, may I take up this issue?

**Comment by Manoramsharma** [comment]
> @Manoramsharma Unassigning due to inactivity. We need to give others a chance.

@palisadoes I have figured out the approach for this there was a blocker for its implementation in the current logic that I too discussed with @xoldyckk , also this needs few considerations. I will be doing it soon.

**Comment by Manoramsharma** [comment]
> Okay seems like Block/Unblock has been implemented in the backend, may I take up this issue?

its not about the implementation but the options for block/unblock should be rendered accordingly only for the members of the organisation instead currently all the users are being fetched there should be a filter function for users to be present in memberId list then block/unblock functionality can be implemented

**Comment by palisadoes** [comment]
Assigning to @prathamesh-mutkure as he requested assignment first as is our policy

**Comment by faraz16iqbal** [comment]
@palisadoes can I work on this issue side by side as well? I have a very easy and intuitive method in mind

**Comment by palisadoes** [comment]
> @palisadoes can I work on this issue side by side as well? I have a very easy and intuitive method in mind

One person per issue. No duplicate issues. Please read the CONTRIBUTING.md for details. We only reassign after 10 days of inactivity on an issue.

**Comment by satyamksharma** [comment]
Hi there as I can see there is no activity so I request to be assigned.

**Comment by tasneemkoushar** [comment]
@prathamesh-mutkure , any progress?

**Comment by prathamesh-mutkure** [comment]
Hi @tasneemkoushar, it's just 3 days since I've been assigned the issue

I'll submit the PR by Sunday

**Comment by prathamesh-mutkure** [comment]
The unblock mutation seems to be broken, will wait for that to be merged before submitting PR for this

Issue - https://github.com/PalisadoesFoundation/talawa-api/issues/1169

**Comment by palisadoes** [comment]
You can proceed with this as the other issue has been merged

**Comment by prathamesh-mutkure** [comment]
Hi @tasneemkoushar @palisadoes 

I've patched the backend for blocking/unblocking functionality, and the frontend seems to be working fine now

- Admin is able to block users
- Blocked users can be unblocked

One more requirement you mentioned in the issue -

- Admin should be able to get a list of blocked members

We can already see if the user is blocked on the existing page, do we need another page to see a list of users who have been blocked?

**Comment by palisadoes** [comment]
Unassigning due to inactivity and no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by Manoramsharma** [comment]
@palisadoes please assign this to me. There are few considerations implementing this, as discussed here https://thepalisadoes-dyb6419.slack.com/archives/CSWH4PN0M/p1678636151401999 
@xoldyckk how are we proceeding for this?

**Comment by prathamesh-mutkure** [comment]
Hi @palisadoes, can I take this up?

**Comment by palisadoes** [comment]
Reassigning due to inactivity
